dist/scss/60_components/_c-myaccount.scss

```diff
@@ -0,0 +1,23 @@
+@import '../10_settings/__all';
+@import '../20_tools/__all';
+
+.c-myaccount-header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 $global-spacing-unit-small;
+  margin: $global-spacing-unit-small 0;
+
+  &__title {
```

Comment:
we've been moving away from string concatenation in the scss for shift-react-components. This article explains why pretty well https://csswizardry.com/2017/02/code-smells-in-css-revisited/. Its fine to use `&` for modifiers though!